src/modules/worlds/sagas.ts

```diff
@@ -2,46 +2,184 @@ import { call, put, select, takeEvery } from 'redux-saga/effects'
+    const { worldName } = action.payload
+    const worldPremissions: WorldPermissions | null = yield call(WorldsAPIContent.getPermissions, worldName)
+    if (!worldPremissions) {
+      return
+    }
+
+    let newWallets: string[] = []
 
-  if (!address) {
-    return
+    if (worldPremissions.access.type === WorldPermissionType.AllowList) {
+      newWallets = [...newWallets, ...worldPremissions.access.wallets]
+    }
+    if (worldPremissions.deployment.type === WorldPermissionType.AllowList) {
+      newWallets = [...newWallets, ...worldPremissions.deployment.wallets]
+    }
+    if (worldPremissions.streaming.type === WorldPermissionType.AllowList) {
+      newWallets = [...newWallets, ...worldPremissions.streaming.wallets]
+    }
+
+    if (newWallets.length > 0) {
+      yield put(getProfilesRequest(newWallets))
+    }
+
+    yield put(getWorldPermissionsSuccess(worldName, worldPremissions))
   }
 
-  yield put(fetchWorldsWalletStatsRequest(address))
+  function* handlePostWorldPermissionsRequest(action: PostWorldPermissionsRequestAction) {
+    const { worldName, worldPermissionNames, worldPermissionType } = action.payload
+    try {
+      const worldPremissions: boolean | null = yield call(
+        WorldsAPIContent.postPermissionType,
+        worldName,
+        worldPermissionNames,
+        worldPermissionType
+      )
+
+      if (!worldPremissions) {
+        return
+      }
+
+      yield put(postWorldPermissionsSuccess(worldName, worldPermissionNames, worldPermissionType))
+    } catch (e) {
+      yield put(
+        postWorldPermissionsFailure(
+          worldName,
+          worldPermissionNames,
+          worldPermissionType,
+          isErrorWithMessage(e) ? e.message : 'Unknown error'
+        )
+      )
+    }
+  }
+
+  function* handlePutWorldPermissionsRequest(action: PutWorldPermissionsRequestAction) {
+    const { worldName, worldPermissionNames, worldPermissionType, newData } = action.payload
+    try {
+      const worldPremissions: boolean | null = yield call(WorldsAPIContent.putPermissionType, worldName, worldPermissionNames, newData)
+
+      if (!worldPremissions) {
+        return
+      }
+
+      yield put(putWorldPermissionsSuccess(worldName, worldPermissionNames, worldPermissionType, newData))
+      yield put(getProfilesRequest([newData]))
+    } catch (e) {
+      yield put(
+        putWorldPermissionsFailure(
+          worldName,
+          worldPermissionNames,
+          worldPermissionType,
+          newData,
+          isErrorWithMessage(e) ? e.message : 'Unknown error'
+        )
+      )
+    }
+  }
+
+  function* handleDeleteWorldPermissionsRequest(action: DeleteWorldPermissionsRequestAction) {
+    const { worldName, worldPermissionNames, worldPermissionType, address } = action.payload
+    try {
+      const worldPremissions: boolean | null = yield call(WorldsAPIContent.deletePermissionType, worldName, worldPermissionNames, address)
```

Comment:
Should this variable be named differently to represent the result of the api call?
```suggestion
      const hasDeletedPermissions: boolean | null = yield call(WorldsAPIContent.deletePermissionType, worldName, worldPermissionNames, address)
```
The same goes for the other sagas here.